vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/vaadin-grid-flow-selection-column.js

```diff
@@ -24,11 +24,11 @@ import { GridColumnElement } from '@vaadin/vaadin-grid/src/vaadin-grid-column.js
       return {
 
         /**
-         * Width of the cells for this column.
+         * Set autoWidth is true.
          */
-        width: {
-          type: String,
-          value: '58px'
```

Comment:
Should we still keep this `58px` value as the default? In 99% of the cases it's still a better guess than the `100px` value derived from `vaadin-grid-column` otherwise. The jump that occurs on auto-width might be smaller with this value.

Reply:
yes, it should not replace harded-value width (58px) with auto-width as the default.
But I think the width of flow selection column should adapt to width of checkbox.
So I have new 2 methods: setWidth and setAutoWidth for the flow-selection-column, and we can set width and auto-width.

Reply:
No new methods (features) in a bugfix. A user doesn't have access to the selection column and thus can't set an explicit width for it so we're not breaking any behavior by making auto-width the default here, right?

Reply:
yes, got it. no feature for fixing bug and no breaking any behavior by adding new features.  I understand it after asking.